bulk/sensor_bulk_load-AssetRecord.csv

```diff
@@ -1629,6 +1629,8 @@ CGINS-METLGR-00035,,Sensor,0,Bulk Meteorology Instrument Package: METBK LGR Modu
 CGINS-METLGR-00036,,Sensor,0,Bulk Meteorology Instrument Package: METBK LGR Module,METBK LGR MODULE,Star Engineering,ASIMET,LGR036,,,,,Woods Hole Oceanographic Institution
 CGINS-METLGR-00037,,Sensor,0,Bulk Meteorology Instrument Package: METBK LGR Module,METBK LGR MODULE,Star Engineering,ASIMET,LGR037,,,,,Woods Hole Oceanographic Institution
 CGINS-METLGR-00038,,Sensor,0,Bulk Meteorology Instrument Package: METBK LGR Module,METBK LGR MODULE,Star Engineering,ASIMET,LGR038,,,,,Woods Hole Oceanographic Institution
+CGINS-METLGR-99991,,Sensor,0,Bulk Meteorology Instrument Package: METBK LGR Module,METBK LGR MODULE,Star Engineering,ASIMET,N/A,,,,Placeholder for unknown S/N (METBK 1), Woods Hole Oceanographic Institution
+CGINS-METLGR-99992,,Sensor,0,Bulk Meteorology Instrument Package: METBK LGR Module,METBK LGR MODULE,Star Engineering,ASIMET,N/A,,,,Placeholder for unknown S/N (METBK 2), Woods Hole Oceanographic Institution
```

Comment:
CGINS-METLGR-99991 and CGINS-METLGR-99992 do not match the SN's used on the cal csv's. Which are correct?

Reply:
I believe the -9999x UIDs are to be used when the METLGR S/N was not captured/documented.

Reply:
The csv's though, are named -00091 and -00092. Should they match what's in this file?

Reply:
The UIDS should be CGINS-METLGR-99991 and CGINS-METLGR-99992.  I was going to do just one (CGINS-METLGR-99999) but because some of the CSV are values depending on if the unit is METBK1 or METBK2, I need to discriminate between them.